### PR TITLE
Schedule CI to run 10 times on a Saturday morning

### PR DESCRIPTION
**Pull request by andrewlock**
## Summary of changes

- Add an AzDo schedule that runs CI 10 times on a Saturday morning
- Always run the exploration tests on these builds

## Reason for change

We're struggling with flaky tests at the moment, so we want more data about what's failing.

## Implementation details

Setup a schedule to start a CI run on master, every 6mins, starting at 1am on Saturday. That gives us 10 runs.

## Test coverage

We can find out if it worked in a week!

## Other details
[Crontab expression reference](https://github.com/atifaziz/NCrontab/wiki/Crontab-Expression), [AzDo schedule docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml)

### CODE REVIEW

**Review by OmerRaviv — APPROVED** [review]
Thank you!

File: .azure-pipelines/ultimate-pipeline.yml [hunk 1]
```diff
@@ -83,7 +83,13 @@ schedules:
         - /benchmarks/*
       exclude:
         - release/1.x
+    always: true
 
+  - cron: "*/6 1 * * Sat"
+    displayName: Saturday CI, every 6 minutes from 1am
```
Comment (pierotibou) [inline .azure-pipelines/ultimate-pipeline.yml:89]: what about the polar bears ? :)

**Review by pierotibou — APPROVED** [review]

File: .azure-pipelines/ultimate-pipeline.yml (hunk 1 above)
Comment (andrewlock) [reply .azure-pipelines/ultimate-pipeline.yml:89]: Well, we're running the VMs either way, may as well use them for something 😉

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #3202 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e): 
* 1 benchmarks are **slower**, with geometric mean 1.243
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`WriteAndFlushEnrichedTraces`|net472|715μs|547ns|2.12μs|0.359| 0|0|3.18&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|472μs|177ns|663ns|0| 0|0|2.59&nbsp;KB|
|#3202|`WriteAndFlushEnrichedTraces`|net472|717μs|589ns|2.28μs|0.357| 0|0|3.18&nbsp;KB|
|#3202|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|457μs|387ns|1.5μs|0| 0|0|2.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AppSecBodyBenchmark</strong> - Slower :warning: Same allocations :heavy_check_mark:



#### Slower :warning: in #3202

|  Benchmark         | diff/base | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.AppSecBodyBenchmark.AllCycleSimpleBody&#8209;net472 | 1.243 | 185.35 | 230.38 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`AllCycleSimpleBody`|net472|186ns|0.242ns|0.938ns|0.0676| 0|0|425&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`AllCycleSimpleBody`|netcoreapp3.1|236ns|0.328ns|1.27ns|0.00576| 0|0|424&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`AllCycleMoreComplexBody`|net472|187ns|0.163ns|0.587ns|0.0638| 0|0|401&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`AllCycleMoreComplexBody`|netcoreapp3.1|232ns|0.204ns|0.791ns|0.00548| 0|0|400&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`BodyExtractorSimpleBody`|net472|260ns|0.212ns|0.821ns|0.0573| 0|0|361&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`BodyExtractorSimpleBody`|netcoreapp3.1|224ns|0.169ns|0.634ns|0.00363| 0|0|272&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`BodyExtractorMoreComplexBody`|net472|14.7μs|12.3ns|44.4ns|1.2| 0.0147|0|7.62&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`BodyExtractorMoreComplexBody`|netcoreapp3.1|12.1μs|20.7ns|77.5ns|0.0896| 0|0|6.75&nbsp;KB|
|#3202|`AllCycleSimpleBody`|net472|230ns|0.207ns|0.801ns|0.0676| 0|0|425&nbsp;B|
|#3202|`AllCycleSimpleBody`|netcoreapp3.1|237ns|0.27ns|1.05ns|0.00589| 0|0|424&nbsp;B|
|#3202|`AllCycleMoreComplexBody`|net472|185ns|0.18ns|0.699ns|0.0637| 0|0|401&nbsp;B|
|#3202|`AllCycleMoreComplexBody`|netcoreapp3.1|233ns|0.406ns|1.62ns|0.00551| 0|0|400&nbsp;B|
|#3202|`BodyExtractorSimpleBody`|net472|258ns|0.205ns|0.795ns|0.0573| 0|0|361&nbsp;B|
|#3202|`BodyExtractorSimpleBody`|netcoreapp3.1|224ns|0.221ns|0.767ns|0.00372| 0|0|272&nbsp;B|
|#3202|`BodyExtractorMoreComplexBody`|net472|14.8μs|10.6ns|41.1ns|1.21| 0.0148|0|7.62&nbsp;KB|
|#3202|`BodyExtractorMoreComplexBody`|netcoreapp3.1|12.1μs|15.2ns|59ns|0.0911| 0|0|6.75&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Unknown :shrug: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendRequest`|netcoreapp3.1|182μs|223ns|864ns|0.273| 0|0|20.57&nbsp;KB|
|#3202|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#3202|`SendRequest`|netcoreapp3.1|182μs|437ns|1.69μs|0.272| 0.0907|0|20.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`ExecuteNonQuery`|net472|1.85μs|1.21ns|4.67ns|0.15| 0.000919|0|947&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`ExecuteNonQuery`|netcoreapp3.1|1.35μs|0.576ns|2.15ns|0.013| 0|0|936&nbsp;B|
|#3202|`ExecuteNonQuery`|net472|1.89μs|2.14ns|8.27ns|0.15| 0.000946|0|947&nbsp;B|
|#3202|`ExecuteNonQuery`|netcoreapp3.1|1.43μs|1.55ns|5.99ns|0.0126| 0.00074|0|936&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`CallElasticsearch`|net472|2.54μs|0.905ns|3.5ns|0.183| 0|0|1.16&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`CallElasticsearch`|netcoreapp3.1|1.55μs|0.483ns|1.81ns|0.0146| 0|0|1.1&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`CallElasticsearchAsync`|net472|2.59μs|0.533ns|2.07ns|0.205| 0|0|1.29&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`CallElasticsearchAsync`|netcoreapp3.1|1.7μs|0.476ns|1.78ns|0.0161| 0|0|1.22&nbsp;KB|
|#3202|`CallElasticsearch`|net472|2.59μs|1.93ns|7.24ns|0.183| 0|0|1.16&nbsp;KB|
|#3202|`CallElasticsearch`|netcoreapp3.1|1.57μs|0.573ns|2.06ns|0.0141| 0|0|1.1&nbsp;KB|
|#3202|`CallElasticsearchAsync`|net472|2.78μs|1.57ns|5.67ns|0.205| 0|0|1.29&nbsp;KB|
|#3202|`CallElasticsearchAsync`|netcoreapp3.1|1.59μs|1.02ns|3.69ns|0.0158| 0|0|1.22&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`ExecuteAsync`|net472|2.64μs|4.19ns|15.7ns|0.226| 0|0|1.42&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`ExecuteAsync`|netcoreapp3.1|1.71μs|2.64ns|10.2ns|0.0179| 0|0|1.34&nbsp;KB|
|#3202|`ExecuteAsync`|net472|2.61μs|7.08ns|27.4ns|0.225| 0|0|1.42&nbsp;KB|
|#3202|`ExecuteAsync`|netcoreapp3.1|1.73μs|3.17ns|11.8ns|0.018| 0|0|1.34&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendAsync`|net472|5.69μs|13.6ns|52.6ns|0.438| 0|0|2.77&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendAsync`|netcoreapp3.1|3.6μs|8.06ns|31.2ns|0.0359| 0|0|2.6&nbsp;KB|
|#3202|`SendAsync`|net472|5.66μs|12.8ns|49.4ns|0.438| 0|0|2.77&nbsp;KB|
|#3202|`SendAsync`|netcoreapp3.1|3.61μs|11.6ns|44.9ns|0.034| 0|0|2.6&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|net472|3.21μs|1.47ns|5.67ns|0.287| 0|0|1.81&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|netcoreapp3.1|2.45μs|0.701ns|2.62ns|0.0258| 0|0|1.85&nbsp;KB|
|#3202|`EnrichedLog`|net472|3.24μs|3.57ns|13.8ns|0.287| 0|0|1.81&nbsp;KB|
|#3202|`EnrichedLog`|netcoreapp3.1|2.57μs|1.22ns|4.4ns|0.0243| 0|0|1.85&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|net472|151μs|47.4ns|184ns|0.678| 0.226|0|4.65&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|netcoreapp3.1|115μs|75.7ns|293ns|0.0574| 0|0|4.49&nbsp;KB|
|#3202|`EnrichedLog`|net472|151μs|217ns|842ns|0.678| 0.226|0|4.66&nbsp;KB|
|#3202|`EnrichedLog`|netcoreapp3.1|116μs|143ns|536ns|0.0593| 0|0|4.49&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|net472|5.55μs|9.07ns|32.7ns|0.569| 0.00277|0|3.59&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|netcoreapp3.1|4.26μs|6.92ns|25.9ns|0.0527| 0|0|3.91&nbsp;KB|
|#3202|`EnrichedLog`|net472|5.59μs|10.7ns|41.4ns|0.569| 0.00279|0|3.59&nbsp;KB|
|#3202|`EnrichedLog`|netcoreapp3.1|4.27μs|5.52ns|19.9ns|0.0525| 0|0|3.91&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendReceive`|net472|2.35μs|1.92ns|7.17ns|0.217| 0|0|1.37&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`SendReceive`|netcoreapp3.1|1.87μs|2.08ns|7.78ns|0.0179| 0|0|1.32&nbsp;KB|
|#3202|`SendReceive`|net472|2.36μs|1.8ns|6.72ns|0.218| 0|0|1.37&nbsp;KB|
|#3202|`SendReceive`|netcoreapp3.1|1.8μs|1.01ns|3.93ns|0.018| 0|0|1.32&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|net472|5.03μs|2.5ns|9.67ns|0.352| 0|0|2.23&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`EnrichedLog`|netcoreapp3.1|4.35μs|1.79ns|6.7ns|0.0239| 0|0|1.8&nbsp;KB|
|#3202|`EnrichedLog`|net472|5.16μs|3.43ns|13.3ns|0.353| 0|0|2.23&nbsp;KB|
|#3202|`EnrichedLog`|netcoreapp3.1|4.43μs|2.11ns|8.15ns|0.0245| 0|0|1.8&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`StartFinishSpan`|net472|1.12μs|0.793ns|2.97ns|0.128| 0|0|810&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`StartFinishSpan`|netcoreapp3.1|997ns|0.211ns|0.791ns|0.0102| 0|0|760&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`StartFinishScope`|net472|1.42μs|0.546ns|2.04ns|0.141| 0|0|891&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`StartFinishScope`|netcoreapp3.1|1.08μs|0.76ns|2.94ns|0.0118| 0|0|880&nbsp;B|
|#3202|`StartFinishSpan`|net472|1.19μs|0.762ns|2.85ns|0.128| 0|0|810&nbsp;B|
|#3202|`StartFinishSpan`|netcoreapp3.1|925ns|0.905ns|3.39ns|0.0102| 0|0|760&nbsp;B|
|#3202|`StartFinishScope`|net472|1.4μs|0.793ns|3.07ns|0.141| 0|0|891&nbsp;B|
|#3202|`StartFinishScope`|netcoreapp3.1|1.05μs|0.331ns|1.28ns|0.0121| 0|0|880&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.TraceAnnotationsBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`RunOnMethodBegin`|net472|1.51μs|1.17ns|4.53ns|0.141| 0|0|891&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/4cfe6483fef6fc3a2432fff1302aa54ef7c6369e)|`RunOnMethodBegin`|netcoreapp3.1|1.12μs|0.527ns|1.97ns|0.0118| 0|0|880&nbsp;B|
|#3202|`RunOnMethodBegin`|net472|1.56μs|0.989ns|3.83ns|0.141| 0|0|891&nbsp;B|
|#3202|`RunOnMethodBegin`|netcoreapp3.1|1.23μs|0.67ns|2.6ns|0.0117| 0|0|880&nbsp;B|
</details>

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #3202 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/65c6c4e95b8092d1e4608d482f5e91a8044eaf42) will **not change** line coverage
:heavy_check_mark: Merging #3202 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/65c6c4e95b8092d1e4608d482f5e91a8044eaf42) will **not change** branch coverage
:heavy_check_mark: Merging #3202 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/65c6c4e95b8092d1e4608d482f5e91a8044eaf42) will **not change** complexity


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/65c6c4e95b8092d1e4608d482f5e91a8044eaf42) | #3202       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `17971` / `24517` | `17936` / `24517` |          |
| Lines %   | `73%`      | `73%`      |  `0%` :heavy_check_mark:  |
| Branches  | `10585` / `15050` | `10582` / `15050` |          |
| Branches %| `70%`      | `70%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `16261`      | `16261`        |  `0`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=111938&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/111938/artifacts?artifactName=Code%20Coverage%20Report_111938&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3202](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=111847&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/111847/artifacts?artifactName=Code%20Coverage%20Report_111847&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/65c6c4e95b8092d1e4608d482f5e91a8044eaf42) | #3202       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `73%`      | `73%`       |  `0%` :heavy_check_mark:  |
| Branches %| `70%`      | `70%`       |  `0%` :heavy_check_mark:  |
| Complexity| `16261`      | `16261`       |  `0` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.Ci.GitInfo](https://github.com/DataDog/dd-trace-dotnet/tree/8361cffb8223f3a69e7084dd7f9695f26a42a919/tracer/src/Datadog.Trace/Ci/GitInfo.cs) | `-17%` :no_entry: | `-11%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Ci.CIVisibility](https://github.com/DataDog/dd-trace-dotnet/tree/8361cffb8223f3a69e7084dd7f9695f26a42a919/tracer/src/Datadog.Trace/Ci/CIVisibility.cs) | `5%` :heavy_check_mark: | `6%` :heavy_check_mark:   | `0` :heavy_check_mark: |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=111938&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/111938/artifacts?artifactName=Code%20Coverage%20Report_111938&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3202](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=111847&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/111847/artifacts?artifactName=Code%20Coverage%20Report_111847&api-version=6.0&%24format=zip)

**Pull request merged**